ios/Podfile

```diff
@@ -14,5 +14,4 @@ end
 
 target 'PluginTests' do
   capacitor_pods
-  pod 'Intercom', '~> 11.0'
```

Comment:
Why was this removed?

Reply:
This is simply for the 'PluginTests' target which we're not leveraging at the moment, this line has moved to the 'Plugin' target which is correct